source/chapters/djing_with_mixxx.rst

```diff
@@ -352,3 +352,130 @@ DJ as follows:
+
+Mixing With Intro & Outro Cues
+------------------------------
+
+You can line up the ends of the intro and outro so the energy of the new song picks up
+right as the old track ends.
+
+.. figure:: ../_static/ends-aligned-waveforms.png
+   :align: center
+   :alt: Intro and outro ends aligned
+   :figclass: pretty-figures
+
+   Intro and outro end markers aligned showing on the waveform
+
+If the intro and outro are short enough to fit on the waveform at the same time, you can simply watch the
+waveforms and press play when the markers align. Otherwise, you can watch the time remaining on the deck, hover
+your mouse cursor over the outro end line on the overview waveform. Mixxx will show you how much time is left
+until that point. When that time equals the length of the intro, press play to precisely line up the end of the
+intro. If you have changed the tempo of the track, the time remaining scales to show the real time remaining at the
+adjusted tempo.
+
+Alternatively, you can line up the start of the intro & outro. In this case, watch the scrolling waveform when the
+outro start point is coming up on the old track. Then, press play on the new track when the old track reaches the outro start.
+
+   .. note:: There are no rules on which method should be used. The method anyone decides to use is an artistic judgement based
+             on the musical content of each track, what the vibe of the crowd is at that moment, and what you want to do with the mix.
+
+If you do not have the intro end and outro start points marked when loading a track, you can find and
+mark these in headphones before you mix in the track. Then seek back to the intro start to get the track ready. if you don't
+have time to find the outro end point before mixing in the track, you can use the new deck cloning feature to drag and drop to a new deck,
+then seek ahead on the new deck and listen in headphones to mark the outro start point. If you don't have time to do that either,
+you can right click on the overview waveform where you think the outro starts. Then look for how far that is from the end of the track to
+compare it to the length of the intro of the next track.
+
+.. figure:: ../_static/outro-right-click.png
+   :align: center
+   :alt: Outro right click
+   :figclass: pretty-figures
+
+   Here the outro is approximately 30 seconds long. However, the analyzer placed the outro end point before the end of the track,
+   so the outro is actually a little bit shorter than 30 seconds. Without marking the outro start point, you would have to do some
+   math in your head to find exactly how long the outro is.
+
+Intro and Outro Cues With AutoDJ
+--------------------------------
+
+Intro & outro section cues can be used to tell AutoDJ how long to crossfade and how to align the tracks in time. AutoDJ uses
+the intro & outro cues in the new "Full Intro + Outro" and "Fade At Outro Start" modes. In both modes, instead of crossfading
```

Comment:
```suggestion
the intro & outro cues in the "Full Intro + Outro" and "Fade At Outro Start" modes. In both modes, instead of crossfading
```
Here again if we call it "new", we'll have to remember to remove the "new" later.